Update Keystore management chapter (#87)
infinum/android-handbook: Common Android/Keystore management.md

The production keystore **must** be protected with strong, random passwords. The keystore file **must** be committed to the repository, but the credentials will be saved in a password manager and downloaded as needed. The credentials **must not** be committed to the repository.

To create a production keystore, you can use Android Studio built-in tool. "Build" -> "Generate Signed Bundle / APK" -> "Next" -> "Create new...". Since the keystore will be a part of the repository, save it somewhere there. The name for the key store should be "package.name.of.your.project".
Then we need to choose a password. Try to use some random password generator for this step and make sure to keep the password safe. This password is used both for keystore and the signing/upload key itself, since if you try to use different passwords you will get an error. Then we need to choose key alias, which can simply be the name of the project. The last thing you need to fill out is at least one of the fields under "Certificate" section. You can put "Infinum" at "Organization" for example.

![Creation of a new keystore file](/img/create_keystore_example.png)

> After a successful creation of a keystore file and password, send the password and key alias to TL so it gets saved to a password manager. Commit the keystore file to the repository and push it to remote. If Vault is already setup for your project, then add password and key alias to it as well.

Author: > send the keystore file

Hmm... Do we really need to send the keystore file? What will the TL do with it? 😄 

> and all credentials

I would rather leave it as "send password and key alias" so we exactly know what to send. If we leave it as "all credentials" I can already see some confusion and unnecessary wasting of time 🤷

> After a successful creation of a keystore file and password, send the password and key alias to TL so it gets saved to a password manager. Commit the keystore file to the repository and push it to remote. If Vault is already setup for your project, then add password and key alias to it as well.

Author: Checkout my suggestion below :)

> After a successful creation of a keystore file and password, send the password and key alias to TL so it gets saved to a password manager. Commit the keystore file to the repository and push it to remote. If Vault is already setup for your project, then add password and key alias to it as well.

Author: ```suggestion
After a successful creation of a keystore file and password, send the password and key alias to TL so it gets saved to our password manager. Commit the keystore file to the repository and push it to remote. For the credentials, add them to the project Vault as well.
```
